src/karts/kart.cpp

```diff
@@ -1710,18 +1715,25 @@ void Kart::updateNitro(float dt)
     bool increase_speed = (m_controls.m_nitro && isOnGround());
     if (!increase_speed && m_min_nitro_time <= 0.0f)
     {
+        if(m_nitro_sound->getStatus()==SFXBase::SFX_PLAYING)
```

Comment:
Hi, can you make so that it looks like:
```if (m_nitro_sound->getStatus() == SFXBase::SFX_PLAYING)```

The same with the other ifs, as it makes the code more readable. Thanks.